### DISCUSSION TITLE
Event Font Colour

### DISCUSSION THREAD

**Discussion by metalreign76** [hoangnm/react-native-week-view#311] [Q&A]
Is there a way to have the event text in a color other than white?

I tried to pass the style property in the EventItem object, setting the color property within but it had no effect.

**Comment by metalreign76** [comment]
I havent been able to make any changes to the font in the individual events displays...weight, size.

**Comment by pdpino** [comment]
Hi @metalreign76, sorry the late reply

You can provide your own [`EventComponent`](https://hoangnm.github.io/react-native-week-view/guides/custom-components#event-content) to customize this, something like this:
```js
const MyEventComponent = (event) => (
  <Text style={[
    { fontWeight: 'bold', fontSize: '20' }, // style for all events
    event.textStyle, // custom style per event, if needed
  ]}>
    {event.description}
  </Text>
)

// ... then provide the prop:
EventComponent={MyEventComponent}
```

**Reply by metalreign76** [reply]
Much obliged and no need to apologise. Thank you for making such a useful component available. I'll try what you suggest. Thanks again.

**Comment by pdpino** [comment]
Also, I think we can add a prop `eventTextStyle` and a special field `event.textStyle`, to make this more straightforward :thinking: 

UPDATE: I added these things in #313
